src/main/java/org/liquidengine/legui/system/renderer/nvg/util/NvgShapes.java

```diff
@@ -20,6 +22,8 @@
  */
 public class NvgShapes {
 
+    public static final Vector4f ZERO_CORNDERS = new Vector4f(0);
```

Comment:
This should be using an interface to make the vector unmodifiable, otherwise anyone change modify this constant.

Reply:
Sure, I'll fix that.